management-api-server/src/main/java/com/datastax/mgmtapi/resources/NodeOpsResources.java

```diff
@@ -663,6 +663,75 @@ public Response searchIndexRebuild(
         });
   }
 
+  @POST
+  @Path("/encryption/internode/truststore/reload")
+  @Produces(MediaType.TEXT_PLAIN)
+  @ApiResponse(
+      responseCode = "200",
+      description = "Truststore reloaded successfully",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
+              examples = @ExampleObject(value = "OK")))
+  @Operation(summary = "reload truststore", operationId = "reloadTruststore")
+  public Response reloadTruststore() {
+    return handle(
+        () -> {
+          final String releaseVersion =
+              ResponseTools.getSingleRowStringResponse(
+                  app.dbUnixSocketFile, app.cqlService, CASSANDRA_VERSION_CQL_STRING);
+          if (!releaseVersion.startsWith("4.0.0.68") && !releaseVersion.startsWith("4.0.0.69")) {
+            // rebuilding search index is only possible on DSE
+            return Response.status(Response.Status.BAD_REQUEST)
+                .entity("Reloading the truststore manually is only possible on DSE")
+                .build();
+          }
+
+          app.cqlService.executeCql(
+              app.dbUnixSocketFile, "CALL NodeOps.reloadInternodeEncryptionTruststore()");
+
+          return Response.ok("OK").build();
```

Comment:
suggestion:
If something goes wrong reloading the truststore, the exception is going to be masked in the API call a bit and just return a 500 Internal Server Error, and it will require digging into the Cassandra/DSE logs to figure out what actually happened. It would be better to catch things here and log them before returning a 500.

Reply:
How can I catch an exception from the RPC `NodeOps.reloadInternodeEncryptionTruststore()`? I don't think exceptions can be passed through the boundary of the RPC call? 

Are you proposing that we return some string from `NodeOps.reloadInternodeEncryptionTruststore()` stating whether the operation errored?

Reply:
Looking at the other endpoint implementations, it looks like many do not document the generic 500 error in these cases, so let's just leave it as is.